repos/system_upgrade/el7toel8/actors/migratentp/libraries/migratentp.py

```diff
@@ -80,12 +85,28 @@ def migrate_ntp(migrate_services, config_tgz64):
 
     step_tickers = '/etc/ntp/step-tickers' if 'ntpdate' in migrate_configs else ''
 
-    ignored_lines = ntp2chrony('/', ntp_conf, step_tickers)
+    migrated, ignored_lines = ntp2chrony('/', ntp_conf, step_tickers)
 
     config_resources = [reporting.RelatedResource('file', mc) for mc in migrate_configs + [ntp_conf]]
     package_resources = [reporting.RelatedResource('package', p) for p in ['ntpd', 'chrony']]
 
-    if not ignored_lines:
+    if not migrated:
+        reporting.create_report([
+            reporting.Title('{} configuration not migrated to chrony'.format(' and '.join(migrate_configs))),
+            reporting.Summary('ntp2chrony failed'),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups(COMMON_REPORT_TAGS)
+        ] + config_resources + package_resources)
+
+    elif failed_services:
+        reporting.create_report([
+            reporting.Title('{} not migrated to chrony'.format(' and '.join(migrate_configs))),
+            reporting.Summary('Failed to enable {} service'.format(' and '.join(failed_services))),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups(COMMON_REPORT_TAGS)
+        ] + config_resources + package_resources)
```

Comment:
Please fix the reports. This is not report but log.
* TItle is supposed to be statis. When there is a problem and the problem has different parameters (e.g. list of files is different), the problem is still same. So the title is expected to be same. I see that other reports in the file are wrong as well. These should be ideally fixed as well.
* Current summary is not summary but log. When writting summary, it's good to ask yourself questions like:
  * What is the thing I want user to take from this report?
  * How there report helps user to understand the problem and fix it?
  * ....

Reply:
Do you want that to be addressed in a separate PR? This one is about not using StopActorExecutionError.
.

Reply:
It should be addressed in this one as the exception replacement introduces problematic reports. Or you can drop the reports and keep just the logging. There is still plenty of time till CTC1 (10w) so I think it's worthy to do it just once and right.

Btw, regarding systemd services we are going to introduce (today?) new funcionality that handles enabling/disabling of systemd services in general: https://github.com/oamg/leapp-repository/pull/964

Reply:
you could also drop the post-upgrade reports completely which will resolve this problem in this PR. do not forget to update other stuff so tests are passing.

Reply:
Seems good to me. Thanks